src/sagemaker/mxnet/estimator.py

```diff
@@ -21,7 +21,8 @@ class MXNet(Framework):
 
     __framework_name__ = "mxnet"
 
-    def __init__(self, entry_point, source_dir=None, hyperparameters=None, py_version='py2', **kwargs):
+    def __init__(self, entry_point, source_dir=None, hyperparameters=None, py_version='py2',
+                 framework_version=MXNET_VERSION, **kwargs):
```

Comment:
Something to consider: shall the framework_version be part of the Framework class and not be repeated in each of the child classes?

Reply:
Maybe in the future, we decided not to do it as part of this change.